tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -73,18 +92,26 @@ func runVmBackupAndRestore(brCase VmBackupRestoreCase, expectedErr error, update
 		time.Sleep(brCase.InitDelay)
 	}
 
+	// Check if this VM should be running or stopped for this test.
+	// Depend on pre-backup verification function to poll state.
+	if brCase.PowerState == "Stopped" {
+		log.Print("Stopping VM before backup as specified in test case.")
+		err = v.StopVm(brCase.Namespace, brCase.Name)
+		gomega.Expect(err).To(gomega.BeNil())
```

Comment:
Do we need any waits or are stops instant?

Reply:
I have a wait as the pre-backup verification function in the test table. I forget why I thought that made sense.

Reply:
+1 makes sense to me... just didn't connect the dots that there could be waits outside this function.